packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -3,46 +3,51 @@
         :class="[
             $style['c-filterPill'], {
                 [$style['c-filterPill--selected']]: isToggleSelected,
-                [$style['c-filterPill--disabled']]: isToggleDisabled
+                [$style['c-filterPill--disabled']]: isDisabled
             }]"
         data-test-id="filter-item">
-        <a
-            :class="$style['c-filterPill-link']"
-            :href="href"
-            tabindex="-1"
-            aria-hidden="true"
-            data-test-id="filter-pill-link">
-            <span class="is-visuallyHidden">
-                {{ displayText }}
-            </span>
-        </a>
-        <input
-            :id="`filterPillToggle-${inputId}`"
-            type="checkbox"
-            class="is-visuallyHidden"
-            :class="$style['c-filterPill-checkbox']"
-            data-test-id="filter-pill-input"
-            :checked="isToggleSelected"
-            :tabindex="0"
-            :disabled="isToggleDisabled"
-            @change="toggleFilter">
-        <label
-            :class="$style['c-filterPill-label']"
-            :for="`filterPillToggle-${inputId}`"
-            data-test-id="filter-pill-label"
-            :tabindex="-1">
-            <tick-icon :class="$style['c-filterPill-icon']" />
-            <span
-                :class="$style['c-filterPill-text']"
-                data-test-id="filter-pill-text">
-                {{ displayText }}
-            </span>
-            <span
-                :class="$style['c-filterPill-number']"
-                data-test-id="filter-pill-number">
-                {{ displayNumber }}
-            </span>
-        </label>
+        <div v-if="isLoading">
```

Comment:
it might be a part of the styling ticket, but it would be great to extract the loading thing into a separate component.

Reply:
absolutely - I'd suggest we do that as part of the styling work if that's ok :)